pkg/chunk/repair.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package chunk
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"strings"
+
+	"github.com/ethersphere/bee/pkg/api"
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/pss"
+	"github.com/ethersphere/bee/pkg/pushsync"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/bee/pkg/trojan"
+)
+
+const (
+	// RecoveryTopicText is the string used to construct the recovery topic
```

Comment:
please go through all of the definition comments in the diff and add a period (`.`) at the end of each one, so that we follow the coding guidelines

Reply:
at least the following new functions are missing comments:
- `WithPushSyncer` 
- `deliverToPSS`
- `handleDeliveryResponse`

Reply:
Do we have to add comments to un-exported functions too? please point me to the coding guideline for the same.

Reply:
no. we don't have to. but it's better if we do.

not a blocker.